src/tool/tool.js

```diff
@@ -1,9 +1,11 @@
 import crosshair from './crosshair';
 import fibonacciFan from './fibonacciFan';
 import measure from './measure';
+import container from './container';
 
 export default {
     crosshair: crosshair,
     fibonacciFan: fibonacciFan,
+    container: container,
```

Comment:
I wouldn't go hunting here for a container, not sure where I would start mind you...